src/docs/command-line.dox

```diff
@@ -187,7 +187,8 @@ format.
 
 @par <code>-p</code>
 Dump in human-readable format (pretty-print). The \c -p flag is incompatible
-with the \c load command.
+with the \c load command. Can be combined with \c -x In this case raw data elements will be
```

Comment:
`Can be` should be a full complete sentence, ending in a period and adding a subject. `The \c -p flag can be combined with \c -x. In this case...`